script.js

```diff
@@ -0,0 +1,20 @@
+const hamburg = document.querySelector('.menu_for_mbl');
+const overlay = document.querySelector('.open');
+const mainMenu = document.querySelector('.main-menu');
+const cancel = document.querySelector('.cancel');
+
+const port = document.querySelector('.portfolio');
+
+hamburg.addEventListener('click', () => {
+  mainMenu.style.display = 'flex';
+  overlay.style.display = 'block';
+});
+
+cancel.addEventListener('click', () => {
+  overlay.style.display = 'none';
+});
+
+
+port.addEventListener('click', () => {
+  mainMenu.style.backgroundColor = 'none';
+});
```

Comment:
Kindly make sure to fix this issue. :eyes: :eyes:. A project without errors will show to the recruiters your proficiency in coding.
![image](https://user-images.githubusercontent.com/98363075/196317868-5ae106cb-e79d-4395-b763-1a7db11a64cb.png)